pixl_core/src/core/db/models.py

```diff
@@ -52,7 +52,7 @@ class Image(Base):
     accession_number: Mapped[str]
     study_date: Mapped[Date] = mapped_column(Date())
     mrn: Mapped[str]
-    hashed_identifier: Mapped[Optional[str]]
+    pseudo_study_uid: Mapped[Optional[str]]
```

Comment:
You'll also need to create an alembic migration for this to be able to be deployed in production

Reply:
I did? https://github.com/UCLH-Foundry/PIXL/pull/389/files#diff-bf17b0fcdc74b312ccafae120d2152b11b53489f1380d128086ee707e9f81a9e

Reply:
ha oh sorry, I swore I looked for them